home.js

```diff
@@ -0,0 +1,192 @@
+const speakerContainerCard = document.querySelector('.speaker-container-card');
+const featureSpeakerButton = document.querySelector('.Feature_speaker_button');
+// line 10 to 13 selects the nav bar details to be clicked.
+
+const linkAbout = document.querySelector('.link-about');
+const linkProgram = document.querySelector('.link-program');
+const linkJoin = document.querySelector('.link-join');
+const linkSponsor = document.querySelector('.link-sponsor');
+
+/* A media query selector to later adjust the display of speakers on the screen, at max-width of
+768, which is a mobile version of the app */
+
+const mediaQueryList = window.matchMedia('(max-width: 768px)');
+
+// Line 19 to 47 shows the click function behaviour
+
+menuBtn.addEventListener('click', () => {
+  menuBtn.classList.add('hidden');
+  headerNavMobile.classList.remove('hidden');
+});
+
+btnCloseNav.addEventListener('click', () => {
+  menuBtn.classList.remove('hidden');
+  headerNavMobile.classList.add('hidden');
+});
+
+linkAbout.addEventListener('click', () => {
+  headerNavMobile.classList.add('hidden');
+  menuBtn.classList.remove('hidden');
+});
+
+linkProgram.addEventListener('click', () => {
+  headerNavMobile.classList.add('hidden');
+  menuBtn.classList.remove('hidden');
+});
+
+linkJoin.addEventListener('click', () => {
+  headerNavMobile.classList.add('hidden');
+  menuBtn.classList.remove('hidden');
+});
+
+linkSponsor.addEventListener('click', () => {
+  headerNavMobile.classList.add('hidden');
+  menuBtn.classList.remove('hidden');
+});
+
+const invitedSpeakers = [
+  {
+    id: 0,
+    url: './images/Elon-Musk-2010.jpeg',
+    name: 'Elon Musk',
+    role: 'CEO  of  SpaceX and Tesla.',
+    description:
+      ' Elon is a Great Enterpreanure and richest man alive. his company has made gaint strides in space shuttle, and the electric car industry.',
+  },
+  {
+    id: 1,
+    url: './images/jeff-Bezos-2017.jpeg',
+    name: 'Jeff Bezoz',
+    role: 'Founder of Amazon',
+    description:
+      'Jeff was once the richest man on the planet, and that was as a result of his great company AWS which is  a subsidary of Amazon. Presently  a market leader in cloud computing',
+  },
+  {
+    id: 2,
+    url: './images/Microsoft-CEO.jpeg',
+    name: 'Satya Nadella',
+    role: 'CEO at MicroSoft inc',
+    description:
+      'Nadella signature achievement as CEO has seen Microsoft Wholesale adoption of cloud computing. Creating the Azure public cloud business, while sifting most of the company software applications to the cloud',
+  },
+  {
+    id: 3,
+    url: './Images/nvidia-ceo.jpg',
+    name: 'Jensen Huang',
+    role: 'CEO AT NVIDIA.',
+    description:
+      'Jensen helped bring life to te graphics chip industry, and has been the CEO since founding the company in 1993. Today, the company is a leader in the grraphics space',
+  },
+  {
+    id: 4,
+    url: './Images/circleCI.jpeg',
+    name: 'Jim Rose',
```

Comment:
- [ ] Kindly make sure you use the correct path on some of your `Speakers` image to make your page looks professional. Right now some of them are not displayed because of some typo errors.
```JavaScript
// Bad 
{
    id: 1,
    url: './images/nvidia-ceo.jpg',
   ...
}

// Good
{
    id: 3,
    url: './Images/nvidia-ceo.jpg',
   ...
}
```

- [ ] Also make sure your **Speakers list** design sticks as much as possible to the one from the Behance template. Right now, each speaker design misses the background image. You can see the example in the images below.

#### Template Design
![Screenshot from 2022-09-16 07-28-27](https://user-images.githubusercontent.com/83042742/190563415-636f430f-9480-497c-8ff1-beb7647edf7a.png)

#### Your Design
![Screenshot from 2022-09-16 07-26-11](https://user-images.githubusercontent.com/83042742/190563116-efa087ad-8013-4bc2-8d7d-f839297385d1.png)

Reply:
Hello [@tresorsawasawa](https://github.com/tresorsawasawa), the idea was to use a design that personalize the original. I used this one for personalization.